### ISSUE TITLE
Find duplicates function for all records of modules

### ISSUE THREAD

**Issue by Mausino** [salesagility/SuiteCRM#1205]
Will it so great if will one time the find duplicate function for contacts, accounts, targets, leads or others in admin area to identify the all potential duplicates in the SuiteCRM system. 

My idea is, that admin will have possibility to choose which module can scan for potential duplicates and will on admin the choise if start with merge of records or skip them.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Find duplicates function for all records of modules](https://trello.com/c/6tzIi3Tv/107-find-duplicates-function-for-all-records-of-modules)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
